### DISCUSSION TITLE
Importing .csv help request

### DISCUSSION THREAD

**Discussion by PwrSunbucks** [RafaelGB/obsidian-db-folder#479] [Q&A] [answered]
When I try to import a csv file into a blank database nothing happens after clicking Import CSV > select file > open. Are there formatting guidelines for the csv that I am not aware of, I checked the documentation but all it says is, "You can upload a CSV file and turn it to a database using the database menu."

Any guidance would be appreciated.

**Comment by imed-ghomari** [comment]
I didn't use this feature myself, so I assumed it would work without configuration, that's why I didn't add much in the docs.

I have reproduced your issue, so let's wait to see if the maintainer has an explanation for this. If not, you should open a bug request with the csv file and the database folder note you're using.

**Comment by RafaelGB** [comment, marked as answer]
Hi @PwrSunbucks ,
There are just two mandatory requirements for CSV import.

- You have to select a Header as filename input (edit your header name on global settings)
- the headers and info must be separated by commas

If in your case nothing is importing, the problem is probably in the first step.
